client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx

```diff
@@ -45,22 +52,22 @@ class MappedDomainType extends React.Component {
 			return null;
 		}
 
-		const learnMoreLink = ( linksTo ) => (
-			<a href={ linksTo } target="_blank" rel="noopener noreferrer" />
-		);
+		let setupInstructionsMessage;
 		let primaryMessage;
 		let secondaryMessage;
 
 		if ( isSubdomain( domain.name ) ) {
+			setupInstructionsMessage = translate(
+				'Follow these instructions to set up your subdomain mapping:'
+			);
 			primaryMessage = translate(
 				'Your subdomain mapping has not been set up. You need to create the correct CNAME or NS records at your current DNS provider. {{learnMoreLink}}Learn how to do that in our support guide for mapping subdomains{{/learnMoreLink}}.',
 				{
 					components: {
-						strong: <strong />,
-						learnMoreLink: learnMoreLink( MAP_SUBDOMAIN ),
+						link: this.renderLinkTo( MAP_SUBDOMAIN ),
```

Comment:
This should be `learnMoreLink` instead of `link` - otherwise it does not render properly:

<img width="670" alt="Screenshot 2021-03-10 at 15 56 51" src="https://user-images.githubusercontent.com/3392497/110658129-55011800-81b9-11eb-9dd1-a1cc882d26a5.png">